doc/aliases/add_toggle.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+/*
+  Creates a toggleable div that includes
+  the text/code associated to a button.
+*/
+var $getDiv = $('div.newInnerHTML').last();
+var buttonName = $getDiv.html();
+var label = getLabelName(buttonName.trim());
```

Comment:
@mshabunin Sorry for the late response, one of the requirements of the buttons code was to make it general. 

So that a person can use it for example for the Languages:
- C++,  C# and Python; 

Some other could instead do the tutorial for:
- Matlab, C++, Java.

And another person could use for versions or other things:
- Version 1, Version 2, Version 3.

It's not only for code but anything where a toggle button could be useful (languages, versions, videos, images,...).